answers/README.md

```diff
@@ -0,0 +1,165 @@
+# ANSWERS
```

Comment:
Why we are using two different naming conventions for body and response?

Reply:
For frontend part we're using camelCase and body comes from frontend part so that's why it's in camelCase whereas for db keys we're using snake case so using that